# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1356124349 -3600
#      Fri Dec 21 22:12:29 2012 +0100
# Node ID 93516992ee34b4abad8c1b857d7161cc5afed439
# Parent  9854666df787bca33dacabd759d49313817aaecf
javamem.m: adapt to 'boxing' Java doubles

diff --git a/scripts/java/javamem.m b/scripts/java/javamem.m
--- a/scripts/java/javamem.m
+++ b/scripts/java/javamem.m
@@ -65,19 +65,19 @@
 ## 2010-08-25 Corrected text on java memory assignments
 ## 2010-09-05 Further overhauled help text
 
 function jmem = javamem ()
 
   rt = javaMethod ("getRuntime", "java.lang.Runtime");
   rt.gc;
   jvmem = cell (3, 1);
-  jvmem{1} = rt.maxMemory ().doubleValue ();
-  jvmem{2} = rt.totalMemory ().doubleValue ();
-  jvmem{3} = rt.freeMemory ().doubleValue ();
+  jvmem{1} = rt.maxMemory ();
+  jvmem{2} = rt.totalMemory ();
+  jvmem{3} = rt.freeMemory ();
 
   if (nargout == 0)
     printf ("\nJava virtual machine (JVM) memory info:\n");
     printf ("Maximum available memory:        %5d MiB;\n",
             jvmem{1} / 1024 / 1024);
     printf ("   (...running garbage collector...)\n");
     printf ("OK, current status:\n");
     printf ("Total memory in virtual machine: %5d MiB;\n",
